app/assets/stylesheets/common/base/reviewables.scss

```diff
@@ -0,0 +1,221 @@
+.reviewable-title {
+  margin-bottom: 1.5em;
+}
+
+.reviewable-topics {
+  width: 100%;
+
+  tbody {
+    td {
+      padding: 0.5em;
+    }
+  }
+}
+
+.reviewable-filters {
+  background-color: $primary-very-low;
+  padding: 0.5em 1em;
+  margin-bottom: 1em;
+  display: flex;
+
+  .all-filters {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+
+  .reviewable-filter {
+    display: flex;
+    align-items: center;
+    margin-right: 1em;
+
+    label {
```

Comment:
as much as possible I prefer when we avoid styling on html elements, it always causes pain later